pmd-core/src/main/java/net/sourceforge/pmd/renderers/CodeClimateRenderer.java

```diff
@@ -35,6 +35,7 @@
 
     // Note: required by https://github.com/codeclimate/spec/blob/master/SPEC.md
     protected static final String NULL_CHARACTER = "\u0000";
+    protected static final List<String> INTERNAL_DEV_PROPERTIES = Arrays.asList("index", "xpath");
```

Comment:
should be `version` and `xpath`

Reply:
:facepalm: Not sure where `index` came from! haha